util-scripts/listening-endpoints/README.md

```diff
@@ -0,0 +1,29 @@
+# Listening Endpoints
+
+This script reports the listening endpoints in the clusters and what processes are listening on them.
+
+**Required Environment Vars:**
+
+* `ROX_ENDPOINT` - Host for StackRox central (central.example.com)
+* `ROX_API_TOKEN` - Token data from [StackRox API token](https://help.stackrox.com/docs/use-the-api/#generate-an-access-token)
```

Comment:
I could not find the API token documentation by following this, because it is redirected to the welcome page of RHACS.

The does not seem to be a section dedicated to creating a token, however, there are procedures for roxctl and CI integration:
- https://docs.openshift.com/acs/3.74/cli/getting-started-cli.html#cli-authentication_cli-getting-started
- https://docs.openshift.com/acs/3.74/integration/integrate-with-ci-systems.html#cli-authentication_integrate-with-ci-systems